Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -118,7 +118,7 @@ The following part explains the configuration file and options:
         Allow: /
         Disallow: /forbidden/
 
-Most settings can also be edited via the backend module `Site Management > Configuration`,
+Most settings can also be edited via the backend module `Site Management > Sites`,
```

Comment:
I would even spell Site Management as it is spelled in the backend (SITE MANAGEMENT) and use textrole `:guilabel:`, but that should probably be checked first how this is done in the rest of the manual and be done consistently. 

See documentation on this:

* [Rules for Referring to GUI Elements](https://docs.typo3.org/typo3cms/HowToDocument/GeneralConventions/ContentStyleGuide.html#rules-for-referring-to-gui-elements)
* [Referring to GUI Elements](https://docs.typo3.org/typo3cms/HowToDocument/GeneralConventions/CodingGuidelines.html#rest-refer-to-gui-elements)

Note that text with the textrole `:guilabel:` is currently not styled at all, but there is an issue for that: [Use specific style for textrole "guilabel"](https://github.com/TYPO3-Documentation/t3SphinxThemeRtd/issues/96)

Reply:
Thank you for the links. I will check out the available textroles, spelling rules etc. before sending more PRs.

Concerning "Site Management", I thought about capitalizing the word but found the present spelling at least in the [Site Handling documentation](https://docs.typo3.org/typo3cms/CoreApiReference/ApiOverview/SiteHandling/Basics.html).